src/platform/forms/save-in-progress/SaveInProgressErrorPage.jsx

```diff
@@ -19,6 +19,15 @@ import { toggleLoginModal } from '../../site-wide/user-nav/actions';
 // For now, this only handles loading errors, but it could feasibly be reworked
 //  to handle save errors as well if we need it to.
 class SaveInProgressErrorPage extends React.Component {
+  componentDidMount() {
+    if (
```

Comment:
@jbalboni @U-DON So this might be a better solution. This does not assume that there is an error string at the end of the url. This is checking if the referrer is coming from auth and load status is 'not attempted'. Since this is the error page component we know that this is an error.

Reply:
Yeah, I think this is better than the way we would have had to change the auth flow.